src/templates/.npmrc

```diff
@@ -0,0 +1,2 @@
+@adobe:registry=https://registry.npmjs.org/
+@testmeshbuilder:registry=https://registry.npmjs.org/
```

Comment:
I was getting 401 unauthorized from artifactory and had to add this to get past that error